### ISSUE TITLE
Translate static/examples into Chinese

### ISSUE THREAD

**Issue by benjaminmeow** [wordplaydev/wordplay#224]
## What's the problem?

Start and finish the static/examples translation into Chinese

## What's the design idea?

This is apart of the Chinese local

## Who benefits?

Chinese speakers, readers, and writers!

## Design specification

See static/examples, and just add a translated name to each bind in each program

**Comment by amyjko** [comment]
@benjaminmeow, is there an assignee for this?

**Comment by awang040403** [comment]
Hi @amyjko, I think static is fully translated. I can work on translating examples into Chinese. Could you please assign me to this?

**Comment by amyjko** [comment]
Sure!

**Comment by awang040403** [comment]
Sounds good! I will check it out, thank you!

**Comment by annieli555** [comment]
[Spring cleaning] @amyjko It's been a few months since this has been assigned, should this be updated?

**Comment by amyjko** [comment]
@awang040403 are you still working on this?

**Comment by awang040403** [comment]
@amyjko Yes, I am still working on this. I completed some of the localization and had some remained. I will commit what I updated soon.

**Comment by amyjko** [comment]
That's great. Thanks @awang040403, let us know how we can help!

**Comment by amyjko** [comment]
Closed in favor of the more general #1310.
